Source/View/LiveCountdownView.swift

```diff
@@ -68,7 +68,9 @@ public class LiveCountdownView: UIView {
     private func initView() {
         Bundle.main.loadNibNamed(LiveCountdownView.nibName, owner: self, options: nil)
         addSubview(contentView)
+        backgroundColor = LIVE_UPDATES_MENU_COLOR
         contentView.frame = bounds
+        contentView.backgroundColor = LIVE_UPDATES_MENU_COLOR
         contentView.autoresizingMask = [.flexibleWidth, .flexibleHeight]
```

Comment:
These <X>_MENU_COLORS are meant for only the main menu buttons and the the views they open